build_cmake/CMakeLists.txt

```diff
@@ -24,7 +24,7 @@ set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
 if (NOT TARGET Framework)
-	add_subdirectory(../../Framework/build_cmake ${CodeGen_SOURCE_DIR}/../../Framework/build_cmake/build)
+	add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/../../Framework/build_cmake ${CMAKE_CURRENT_SOURCE_DIR}/../../Framework/build_cmake/build)
```

Comment:
The TargetArch cmake script is a bit outdated, but it's based on a qt code, the qt code has been updated since so it can be adapted.